src/lib.rs

```diff
@@ -384,67 +439,86 @@ pub fn make_game() -> Result<Game, JsValue> {
     } = get_canvas_info()
         .map_err(|e| JsValue::from(format!("Error initializing canvas: {}", e)))?;
     let ws = WebSocket::new("ws://localhost:3012")?;
-    let game = Game {
-        speed: 2.0,
-        context,
-        width,
-        height,
-        kill_distance: 64.0,
-        local_player_color: Some(Color::random()),
-        players,
-        socket: ws,
+    let wrapper = GameWrapper {
+        game: Arc::new(Mutex::new(Game {
```

Comment:
is `::new` a language-level concept or just a conventional name for part of the implementation?

Reply:
The latter, a conventional name for an otherwise totally normal function